test/powershell/engine/Module/NewModuleManifest.Tests.ps1

```diff
@@ -1,13 +1,73 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+Describe "New-ModuleManifest basic tests" -tags "CI" {
+    BeforeEach {
```

Comment:
Why we need create _the same_ directory for each test?
```suggestion
    BeforeAll {
```
The same for next Describe.